website/www/site/content/en/documentation/sdks/yaml-schema.md

```diff
@@ -0,0 +1,105 @@
+---
+type: languages
+title: "Apache Beam YAML Schema"
+---
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Beam YAML Schema
+
+As pipelines grow in size and complexity, it becomes more common to encounter
+data that is malformed, doesn't meet preconditions, or otherwise causes issues
+during processing.
+
+Beam YAML helps the user detect and capture these issues by using the optional
+`output_schema` configuration, which is available for any transform in the YAML
+SDK. For example, the following code creates a few "good" records and specifies
+that the output schema from the `Create` transform should have records that
+follow the expected schema: `sdk` as a string and `year` as an integer.
+
+```yaml
+pipeline:
+  type: chain
+  transforms:
+    - type: Create
+      config:
+        elements:
+          - {sdk: MapReduce, year: 2004}
+          - {sdk: MillWheel, year: 2008}
+        output_schema:
+          type: object
+          properties:
+            sdk:
+              type: string
+            year:
+              type: integer
+    - type: AssertEqual
+      config:
+        elements:
+          - {sdk: MapReduce, year: 2004}
+          - {sdk: MillWheel, year: 2008}
+```
+
+However, a user will more likely want to detect and handle schema errors. This is where adding an `error_handling` configuration inside the `output_schema` comes into play. For example, the following code will
```

Comment:
If we stay with the current approach, we should call out the difference between normal error handling and schema error handling here.

Reply:
added another paragraph below, thanks